aqua/qt5/Portfile

```diff
@@ -1683,15 +1683,28 @@ foreach {module module_info} [array get modules] {
                 # see https://doc.qt.io/qt-5/qtwebengine-platform-notes.html
                 PortGroup           xcodeversion 1.0
 
-                minimum_xcodeversions   {16 8.3.3}
-                if { ${os.major} < 16 } {
+                # see https://trac.macports.org/ticket/61921
+                minimum_xcodeversions   {17 10.0.0}
+                if { ${os.major} < 17 } {
```

Comment:
this (and the ones below) need the additional `os.platform` check as well

Reply:
I would rather also correct all existing instances in the qt5 portfile first: d26e4d9ed050e467131512330922bc1aeb31f3c9. There remain many instances of this issue in other qt Portfiles; ideally they can be corrected simultaneously in a dedicated PR, and without exacerbating https://trac.macports.org/ticket/64126.